Userland/Applications/VideoPlayer/VideoPlayerWidget.cpp

```diff
@@ -249,7 +249,7 @@ void VideoPlayerWidget::set_time_label(Time timestamp)
 {
     StringBuilder string_builder;
     auto append_time = [&](Time time) {
-        auto seconds = time.to_seconds();
+        auto seconds = time.to_truncated_seconds();
```

Comment:
Does it make sense to round instead of truncate? That's what I did in the web player, seemed like displaying 0.9s as 1s made more sense than displaying it as 0s.

Reply:
Honestly, I think this could go either way. My logic was that it could be considered the truncation of the format "HH:mm:ss.SSS", but since at the moment it's not displaying milliseconds, I don't really have a strong preference.

It looks like Firefox and MPC-BE round to seconds like you did, and Chrome and VLC truncate/floor the seconds.

Reply:
That said, it's probably good if we're consistent in Serenity, obviously!

Reply:
Also if it's easier, we have a digital number formatting helper so you don't have to do manual formatting:
https://github.com/SerenityOS/serenity/blob/master/AK/NumberFormat.h#L28

Then you can do something like:
```c++
auto seconds = round(static_cast<double>(time.to_milliseconds()) / 1000.0); // or floor or whatever
builder.append(human_readable_digital_time(seconds));
```

Reply:
Used that function and changed it to rounding, thanks!